### ISSUE TITLE
About loss curve

### ISSUE THREAD

**Issue by rwbybxx** [lordmartian/deep_avsr#53]
Hello, when I pretrained the AO model, I trained a total of 220 epochs when pertrain_num_words=3. When epoch 190 is reached, val WER is the best, which is 0.259. However, the loss curve seems to be abnormal.
What is the reason?
I uploaded the graph I generated, please help me have a look
Looking forward to your reply, thank you!

![图片1](https://github.com/smeetrs/deep_avsr/assets/168004992/7696372e-a4f0-477a-8f5c-5b3a83df44d4)

![图片2](https://github.com/smeetrs/deep_avsr/assets/168004992/62ce877d-f094-4f02-b973-f82e0aa0a1e4)

**Comment by smeetrs** [comment]
By abnormal, do you mean the presence of large amount of noise in the curve and higher validation loss values compared to training loss?

**Comment by rwbybxx** [comment]
Thank you for your reply!
Yes, the loss of the validation set no longer drops as much as the training set, is this a normal phenomenon?
In addition, this is the loss curve when my PRETRAIN_NUM_WORDS=21, which is also the case.
![11](https://github.com/smeetrs/deep_avsr/assets/168004992/94a652ac-7001-4a11-bd40-7fa3b6382dce)

**Comment by smeetrs** [comment]
To be honest, it's been 4 years since I worked on this project so I don't remember the nature of graphs I had obtained, but I do recall observing some discrepancies in the loss curves. I hadn't debugged them at that time and had proceeded by observing only the WER.

**Comment by rwbybxx** [comment]
Ok, I see. Thank you for your reply!
